Insights/Account/UEBAAnomaly.yaml

```diff
@@ -0,0 +1,88 @@
+SchemaVersion: 1.0
+Type: KQL
+Provider: Sentinel
+DataTypes:
+  - DataType: BehaviorAnalytics
+RequiredInputFieldsSets:
+  - - Account_Name
+    - Account_UPNSuffix
+EntitiesFilter: {}
+BaseQuery: |
+  BehaviorAnalytics | extend UPN = iff(UserPrincipalName contains '#EXT#', replace(@'_', @'@',  tostring(split(UserPrincipalName,'#EXT#')[0])), UserPrincipalName) | where tostring(split(UPN,'@')[0]) =~ '{{Account_Name}}' and tostring(split(UPN,'@')[1])=~ '{{Account_UPNSuffix}}' | extend Evidence = ActivityInsights
+Insights:
+  Id: 145de96e-e63e-47a6-9dc8-1d56736cc68b
+  DisplayName: UEBA Insights
+  Description: |
+    Summarize anomalous user activities - across geographical locations, devices, and environments; across time and frequency horizons (compared to user's own history); as compared to peers' behavior; as compared to organization's behavior.
+  DefaultTimeRange:
+    BeforeRange: 12d
+    AfterRange: 2d
+  ReferenceTimeRange:
+    BeforeRange: 0d
+  SingleValuesQuery: {}
+  TableQuery:
+    ColumnsDefinitions:
+    - Header: Anomaly
+      OutputType: String
+      SupportDeepLink: false
+    - Header: First Time User Anomaly
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: Uncommon Among Peers
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: Uncommon in Tenant
+      OutputType: Number
+      SupportDeepLink: true
+    - Header: First Time Tenant Anomaly
+      OutputType: Number
+      SupportDeepLink: true
+    QueriesDefinitions:
+    - Filter: " where (ActionType in('Add user','Update user','Reset user password') and  (ActivityInsights.FirstTimeUserPerformedAction == true or ActivityInsights.FirstTimeActionPerformedInTenant == true or ActivityInsights.ActionUncommonlyPerformedAmongPeers == true or ActivityInsights.ActionUncommonlyPerformedInTenant == true))"
+      Summarize: summarize ['First Time User Anomaly']=countif(Evidence.FirstTimeUserPerformedAction == true),['Uncommon Among Peers']=countif(Evidence.ActionUncommonlyPerformedAmongPeers == true), ['Uncommon in Tenant']=countif(Evidence.ActionUncommonlyPerformedInTenant == true) , ['First Time Tenant Anomaly']=countif(Evidence.FirstTimeActionPerformedInTenant == True)
```

Comment:
Need to add 'by UPN' at the end, otherwise when we don't have a match, we will get rows with 0 returns which is not how we want no results to show up in the UX.